.github/workflows/publish-package.yml

```diff
@@ -0,0 +1,105 @@
+name: publish distributions
+on:
+  push:
+    branches:
+    - main
+    tags:
+    - v*
```

Comment:
The tags on this repo don't have the v prefix.

Reply:
I'll replace these with a regex later.

Reply:
I pushed a fix for this.

Reply:
Thanks @asmeurer. Sorry I'm currently traveling for work but will get back to this in the next 48 hours.